### DISCUSSION TITLE
v0.54.0 is not installing plugins

### DISCUSSION THREAD

**Discussion by fatihtokus** [aquasecurity/trivy#7272] [Bugs]
### Description

Latest version of trivy v0.54.0 is not installation trivy plugins

### Desired Behavior

Install plugins without an issue

### Actual Behavior

$ curl -s -L -o - https://github.com/aquasecurity/trivy/releases/download/v0.54.0/trivy_0.54.0_Linux-64bit.tar.gz | tar -xz
$ ./trivy plugin install referrer
2024-07-31T10:14:01Z	INFO	[plugin] Updating the plugin index...	url="https://aquasecurity.github.io/trivy-plugin-index/v1/index.yaml"
2024-07-31T10:14:01Z	INFO	[plugin] Installing the plugin...	src="github.com/aquasecurity/trivy-plugin-referrer"

### Reproduction Steps

```bash
1. trivy plugin install referrer
   trivy plugin list
2.
3.
...
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
trivy plugin install referrer
trivy plugin list
```

### Operating System

linux

### Version

```bash
v0.54.0
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
Hello @fatihtokus 
Thanks for your report!

Created #7273 for this task

Regards, Dmitriy
